model/src/lectures/lectureDoc.ts

```diff
@@ -31,16 +31,27 @@ namespace LectureDisplay {
         quizModel: Quizzes.Quiz;
     }
 
+    export interface Question {
```

Comment:
Why not keep track of who asked the question?

Reply:
I had removed it since I wasn't using it for now but I should keep it you are right